source/docs/casper/developers/dapps/sdk/client-library-usage.md

```diff
@@ -271,3 +273,95 @@ print(deploy.hash.hex())
 </Tabs>
 
 Once submitted, the above snippet will print the deploy hash in the console.
+
+## Staking
+
+Token staking is vital to the Casper Network, enabling its delegated Proof-of-Stake consensus mechanism to operate effectively. With the help of any of the Casper SDKs, you can stake tokens by delegating them to validators.
+
+The delegation functionality exists as a piece of session code obtainable from the [casper-node](https://github.com/casper-network/casper-node) repository. To delegate tokens, first clone the repository:
+
+```bash
+git clone https://github.com/casper-network/casper-node.git
+cd casper-node/
+```
+
+Then compile the [delegate contract](https://github.com/casper-network/casper-node/blob/dev/smart_contracts/contracts/client/delegate/src/main.rs):
+
+```bash
+make setup-rs
+make build-contract-rs/delegate
+```
+
+Now, assuming that you cloned `casper-node` from your project's root directory, `cd` back into it:
+
+```bash
+cd ../
```

Comment:
```suggestion
The complied Wasm can be found in this directory: `target/wasm32-unknown-unknown/release/`.

```bash
ls target/wasm32-unknown-unknown/release/delegate.wasm

Reply:
This section is not to show the output directory, but to explain that if they cloned from their project's root directory, they can go back to their root directory and access *delegate.wasm* via the path in the code example. This could be altered but not in the way you've suggested IMO.